test/zero-downtime-flagd-proxy/README.md

```diff
@@ -0,0 +1,28 @@
+# FlagD Proxy Zero downtime test
+
+## How to run
+
+Clone this repository and run the following command:
+
+```shell
+FLAGD_PROXY_IMG=your-flagd-image FLAGD_PROXY_IMG_ZD=your-flagd-second-image ZD_CLIENT_IMG=your-zd-client-image make run-zd-test
+```
+
+This will create a flagd-proxy and a job in `flagd-zd-test` namespace,
+where the test will be run.
+
+Please be aware, you need to build your custom image for the zd-client
+and two images for flagD first.
```

Comment:
I'm a bit confused here. I know how to build my flagd proxy images, but do I build the zd-client image? I don't see a Dockerfile for it.

I'm guessing there's also other prerequisites here? Like having a cluster available, and the images above need to be available to it (ie: a local kind cluster won't have locally built containers available to it).